src/rustdoc.md

```diff
@@ -42,7 +70,7 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
   * If you've used `rustup toolchain link local /path/to/build/$TARGET/stage1`
     previously, then after the previous build command, `cargo +local doc` will
     Just Work.
-* Use `./x.py doc --stage 1 library/std` to use this rustdoc to generate the
+* Use `./x.py doc --stage 1 std` to use this rustdoc to generate the
```

Comment:
This change seems incorrect to me.

Reply:
No, it's valid. I added the support for shorter lib path a while ago. :)